locale/src/locale.cc

```diff
@@ -51,7 +51,11 @@ namespace leatherman { namespace locale {
 
     string translate(string const& s, string const& domain)
     {
-        return boost::locale::translate(s).str(get_locale("", domain));
+        try {
+            return boost::locale::translate(s).str(get_locale("", domain));
+        } catch (exception const&) {
+            return s;
```

Comment:
I also think the exception handling overhead is small enough when we're doing translation work that it's not worth memoizing "we can't translate" and making this function more complicated